### DISCUSSION TITLE
The gtctl how to use local greptime command

### DISCUSSION THREAD

**Discussion by leaf-potato** [GreptimeTeam/greptimedb#4474] [Q&A] [answered]
The `gtctl` is a very useful tool. I want to know how to specify the greptime command instead of downloading it from github. This is very useful during development.

**Comment by killme2008** [comment, marked as answer]
You can specify the local greptime by config file

https://github.com/GreptimeTeam/gtctl/blob/451006b099b559230ced148e0e2a9d42113eaa4f/examples/bare-metal/cluster-with-local-artifacts.yaml#L4

The document to use bare metal mode: https://docs.greptime.com/reference/gtctl#bare-metal

**Reply by leaf-potato** [reply]
Thanks
